junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/ExtensionContext.java

```diff
@@ -173,12 +176,50 @@
 	 * @return the test instance; never {@code null}
 	 * @throws PreconditionViolationException if the test instance is not present
 	 * in this {@code ExtensionContext}
+	 *
+	 * @see #getRequiredTestInstances()
 	 */
+	@API(status = MAINTAINED, since = "5.4")
```

Comment:
I think this has been released prior to 5.4

Reply:
indeed 😉

Reply:
Ditto. Should stay `STABLE`.